pkg/promunifi/udm.go

```diff
@@ -34,9 +34,10 @@ type unifiDevice struct {
 }
 
 func descDevice(ns string) *unifiDevice {
-	labels := []string{"ip", "version", "model", "serial", "type", "mac", "site_name", "name"}
+	labels := []string{"type", "site_name", "name"}
+	infoLabels := []string{"version", "model", "serial", "mac", "ip", "id", "bytes"}
 	return &unifiDevice{
-		Uptime:        prometheus.NewDesc(ns+"uptime", "Uptime", labels, nil),
+		Info:          prometheus.NewDesc(ns+"info", "Device Information", append(labels, infoLabels...), nil),
```

Comment:
Waaaaaaaaaait a sec, what is going on here?

Reply:
fixed in #152